### PR TITLE
Tracklet error parameterization is loaded based on B-field, not per compiler flag anymore

### PR DESCRIPTION
**Pull request by martenole**
- If the tracklets are out of the search road **either** in Y **or** Z they are ignored
- Some additions to the debugging macro

### CODE REVIEW

**Comment by martenole** [comment]
@davidrohr Sorry I just found these changes in a branch of mine, I should have created a pull request about a month ago, but I forgot it when I had to hand in my thesis

File: GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx [hunk 1]
```diff
@@ -148,6 +149,47 @@ bool GPUTRDTracker_t<TRDTRK, PROP>::Init(TRD_GEOMETRY_CONST GPUTRDGeometry* geo)
...
   }
 
+#ifdef GPUCA_ALIROOT_LIB
+  if (mBz < -90) {
+    GPUError("For AliRoot the magnetic field strength needs to be provided externally before initialization");
+    return false;
+  }
+#else
+  mBz = Param().BzkG;
+#endif
```
Comment (davidrohr) [inline GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx:159]: Why cannot you read it from GPUParam in the AliRoot case?

Comment (martenole) [reply GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx]: True, I think I can manually adjust the GPUSettingsEvent.solenoidBz and then also use the GPUParam in AliRoot. I did not see that before. Also the kG to T conversion was not correct :/

**Comment by martenole** [comment]
Yes I did test it thouroughly in AliRoot and this is ready to be merged now

**Pull request merged**